tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/add-operation.bal

```diff
@@ -2,6 +2,12 @@ function intAdd(int a, int b) returns (int) {
     return a + b;
 }
 
+public function overflowByAddition() {
```

Comment:
The function need not to be `public` since it's used in a single file. 
Shall we change these like the other functions?

Reply:
I will look into this.